src/txn/txn_timestamp.c

```diff
@@ -600,14 +600,10 @@ set:	__wt_writelock(session, &txn_global->rwlock);
 
 /*
  * __wt_txn_commit_timestamp_validate --
- *	Validate a timestamp to be not older than running transaction commit
- *	timestamp and running transaction prepare timestamp. Validate a durable
- *	timestamp to be not older than the global oldest and global stable
- *	timestamp.
+ *	Validate the commit timestamp of a transaction.
  */
 int
-__wt_txn_commit_timestamp_validate(WT_SESSION_IMPL *session, const char *name,
-    wt_timestamp_t ts, WT_CONFIG_ITEM *cval, bool durable_ts)
+__wt_txn_commit_timestamp_validate(WT_SESSION_IMPL *session)
```

Comment:
Given that this function is assigning values to the content of the `WT_TXN` structure, I think it should be renamed to reflect that it's more than `validate`